test/int/dbuser_test.go

```diff
@@ -39,49 +49,120 @@ var _ = Describe("AtlasDatabaseUser", func() {
 		Expect(k8sClient.Create(context.Background(), &passwordSecret)).To(Succeed())
 
 		By("Creating the project", func() {
-			createdProject = mdbv1.DefaultProject(namespace.Name, connectionSecret.Name)
+			createdProject = mdbv1.DefaultProject(namespace.Name, connectionSecret.Name).
+				WithIPAccessList(project.NewIPAccessList().WithIP("0.0.0.0/0"))
```

Comment:
note, that for this test to pass we need to set the IP Access Lists (to "allow everyone") to check mongodb connectivity